src/Utilities/Hooks.js

```diff
@@ -67,6 +68,7 @@ export const useHccEnvironmentContext = () => {
   ]);
 
   return {
+    isLightspeedEnable,
```

Comment:
We probably need to add it to the appconstants.js to the IOP_ENVIRONMENT_CONTEXT object there. We import List.js component into the IOP environment so when the 
```
{envContext.isLightspeedEnabled
                            ? ' Lightspeed '
                            : ' Insights '}
```
Check happens, it will throw an error because isLightspeedEnabled is undefined

Reply:
I'll add it there!

Reply:
I don't see any problems after we update the IOP context, should be fine to merge